src/cmd/cli/command/commands.go

```diff
@@ -562,18 +562,43 @@ Generate will write files in the current folder. You can edit them and then depl
 
 		term.Info("Code generated successfully in folder", prompt.Folder)
 
+		// Load the project and check for empty environment variables
+		loader := compose.Loader{ComposeFilePath: filepath.Join(prompt.Folder, "compose.yaml")}
+		project, err := loader.LoadCompose(cmd.Context())
+		if err != nil {
+			return err
```

Comment:
What does it mean when we just generated a project and this fails? We need to have better error handling. Should we do "graceful degradation" and show the old hint (assume no config?)